config/test.exs

```diff
@@ -4,5 +4,3 @@ config :plexy, :logger,
   redactors: [
     {Plexy.Logger.SimpleRedactor, [redact: ["password"], filter: ["secret"]]}
   ]
-
-config :plexy, config: Plexy.TestConfig
```

Comment:
@maxbeizer looks like you've got a failing test from a different part of the test suite. Maybe we should set a default `app_name` for tests?